rocketpy/Rocket.py

```diff
@@ -278,8 +309,138 @@ def __init__(
         # Evaluate static margin (even though no aerodynamic surfaces are present yet)
         self.evaluateStaticMargin()
 
+        self.evaluateDragCoefficient()
+
         return None
 
+    def evaluateDragCoefficient(self):
+        """Calculates and returns the drag coefficient as the sum of
+        of the individual drag coefficients at zero angle of attack
+        as a function of the Mach number. The angle of attack is
+        assumed to be zero.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        self.viscous_friction_coefficient : Function
+            Function of mach number expressing the viscous friction coefficient,
+            defined as in the literature.
+        """
+        self.evaluateViscousFrictionCoefficient()
+        self.evaluateForebodyDragCoefficient()
+
+        self.drag_coefficient_estimate = self.forebody_drag_coefficient + np.sum(
+            [
+                self.aerodynamicSurfaces[i][2]
+                for i in range(len(self.aerodynamicSurfaces))
+            ]
+        )
+
+        if not self.power_off_drag_as_input:
+            self.powerOffDrag = self.drag_coefficient_estimate
+        if not self.power_on_drag_as_input:
+            self.powerOnDrag = self.drag_coefficient_estimate
+
+        return self.drag_coefficient_estimate
+
+    def evaluateViscousFrictionCoefficient(self):
+        """Calculates and returns the viscous friction coefficient as
+        a function of the Mach number. The viscous friction coefficient
+        depends on the Reynolds number and given by a mathematical
+        expression. Some assumptions for calculating this coefficient
+        are that the angle of attack equals zero and the speed of sound
+        is constant and equal to the value passed in init.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        self.viscous_friction_coefficient : Function
+            Function of mach number expressing the viscous friction coefficient,
+            defined as in the literature.
+        """
+
+        def calculations(mach):
+            reynolds = (
+                (self.air_density ** 2)
```

Comment:
@Gui-FernandesBR pointed out that this should be:

```suggestion
            reynolds = (
                (self.air_density)
```

Reply:
We reviewed our references and I believe there is the square, because of an algebraic manipulation. The deduction can be seen below:

![image](https://user-images.githubusercontent.com/52254687/142740651-1d23b880-39de-4d3e-ae90-bc712ab4329c.png)

![image](https://user-images.githubusercontent.com/52254687/142740770-4a2dac42-5d78-4b48-afbb-ffbeadf6a476.png)

Substituting the values yields:

![image](https://user-images.githubusercontent.com/52254687/142740821-e403dd4a-f1a4-4d71-a82c-12a193657161.png)

What do you think, @Gui-FernandesBR @giovaniceotto?

Reply:
Reynolds is a dimensionless quantity... Just change "kinematic" for "dyanamic" in the first image you've sent and it's going to work then, it's a honest and simple mistake.

https://en.wikipedia.org/wiki/Reynolds_number

Reply:
Yes, I still believe there is a slight error here. I hope this source, which has both formulas (using dynamic and kinematic viscosity) can help settle the debate: [https://www.engineeringtoolbox.com/reynolds-number-d_237.html](https://www.engineeringtoolbox.com/reynolds-number-d_237.html)

Reply:
Ok, I understand now what is going on now, apparently our main reference had a typo on the definition of the Reynolds number and we just trusted it. Thank you both for pointing it out and helping us understand the problem. It will be fixed soon.